introduction.md

```diff
@@ -1,2 +1,5 @@
 # Ruby Introduction
 mlen108 here ;]
+
+* Ruby
+* Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?